DESCRIPTION

```diff
@@ -137,6 +140,7 @@ Suggests:
     RSNNS,
     RWeka,
     sda,
+    shapeletLib,
```

Comment:
you cannot add shapelet lib, as long as its not on cran. also please remove all other packages of learners that are not added here